mmcv/utils/config.py

```diff
@@ -275,11 +276,11 @@ def fromstring(cfg_str, file_format):
             # check if users specify a wrong suffix for python
             warnings.warn(
                 'Please check "file_format", the file format may be .py')
-
-        with tempfile.NamedTemporaryFile('w', suffix=file_format) as temp_file:
+        with tempfile.NamedTemporaryFile(
+                'w', suffix=file_format, delete=False) as temp_file:
             temp_file.write(cfg_str)
-            temp_file.flush()
-            cfg = Config.fromfile(temp_file.name)
+        cfg = Config.fromfile(temp_file.name)
+        os.unlink(temp_file.name)
```

Comment:
unlink is a traditional Unix name. Using `os.remove` rather than `os.unlink` may be more readable.